src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -368,6 +368,14 @@ private int constructMergedTable(Table table, List<FeedToMerge> feedsToMerge,
                 Field[] fieldsFoundInZip =
                     table.getFieldsFromFieldHeaders(csvReader.getHeaders(), null);
                 List<Field> fieldsFoundList = Arrays.asList(fieldsFoundInZip);
+                // Initialize future feed's first date to the first calendar date from the validation result.
+                // This is equivalent to either the earliest date of service defined for a calendar_date record or the
+                // earliest start_date value for a calendars.txt record. For MTC, however, they require that GTFS
+                // providers use calendars.txt entries and prefer that this value (which is used to determine cutoff
+                // dates for the active feed when merging with the future) be strictly assigned the earliest
+                // calendar#start_date (unless that table for some reason does not exist).
+                LocalDate futureFeedFirstDate = feedsToMerge.get(0).version.validationResult.firstCalendarDate;
```

Comment:
Is it feedsToMerge(0) or is it feedsToMerge(feedIndex)?

Reply:
It is `feedsToMerge(0)`, but to your point this should probably be moved out of the for loop block.